setup.py

```diff
@@ -45,6 +45,9 @@ def readme():
     ],
     keywords="data",
     install_requires=INSTALL_REQUIRES,
+    entry_points={
+        "console_scripts": ["pydata-google-auth=pydata_google_auth.cli:main"]
+    },
```

Comment:
We only need this if we want to be able to call from the command line without invoking python. Even without this, people can still do `python -m pydata-google-auth`